problem_2/lib/even_fibonacci_numbers.rb

```diff
@@ -0,0 +1,21 @@
+#By considering the terms in the Fibonacci sequence whose 
+#values do not exceed four million, 
+#find the sum of the even-valued terms.
+
+class FibonacciCalculator
+
+	def initialize(limit)
+		@limit = limit
+	end
+
+	def add_fib_sequence
+		fib = [0, 1]
+		place = 0
+			while (fib[place] + fib[place + 1]) < @limit do
+			fib << fib[place] + fib[place + 1]
```

Comment:
pull out `fib[place] + fib[place + 1]` into its own variable because it's used twice and it's easer for readability